README.md

```diff
@@ -147,43 +111,11 @@ The instructions for running with real hardware change often. Please follow:
 * Daqconf wiki: [link](https://github.com/DUNE-DAQ/daqconf/wiki)
 
 ### offline
-Trigger replay app [replay_tp_to_chain](python/trigger/replay_tp_to_chain): A tool for running the trigger application in isolation from the rest of the DUNE DAQ apps. It accepts input file(s) that are simple `.txt`, containing TPs; and runs your configured algorithm over them. These could be real streamed TPs (from the cold-box for example) or simulated ones (LArSoft).
-
-* `python -m trigger.replay_tp_to_chain`: runs the configuration generation code. 
-* `-s X` slows down the input by a factor *X* (by decreasing the clock rate by that factor) which is useful when outputting large logs/debugging information for example. 
-* `--trigger-activity-plugin` specifies the name of the TAM you would like to use. 
-* Similarly `--trigger-candidate-plugin` identifies the TCM. 
-* `--trigger-activity-config` provides a dictionary, which maps the configuration for your specified TAM, triggering thresholds etc. This can be done for the TCM too.
-* `-l` specifies the number of times you would like to play the data file through the system. 
-* Finally, `--input-file` points to your input file of TPs, which have the standard format: 
-<table>
-  <tr>
-    <th> time start </th> 
-    <th> time over threshold </th> 
-    <th> time peak </th> 
-    <th> channel </th> 
-    <th> adc integral </th> 
-    <th> adc peak </th> 
-    <th> det id </th> 
-    <th> type </th>
-  </tr>
-</table> 
-
-Few example files are available at `/nfs/rscratch/trigger/tp_files` at CERN:
-
-* *1000_electron_events.txt*: simulated electron events
-* *tp_dataset_run020472_2s.txt*: real TPs taken from March 2023 Vertical Drift ColdBox, 2 seconds
-* *tp_dataset_run020472_60s.txt*: real TPs taken from March 2023 Vertical Drift ColdBox, 60 seconds
-* *pds_tps_60s.txt*: real TPs from the PDS system
-
-Example command:
-```
-python -m trigger.replay_tp_to_chain -s 10 --trigger-activity-plugin TAMakerHorizontalMuonAlgorithm --trigger-activity-config ‘dict(adjacency_threshold=100, window_length=10000, adj_tolerance=20)‘ --trigger-candidate-plugin TCMakerHorizontalMuonAlgorithm --trigger-activity-config ‘dict(adjacency_threshold=100, window_length=10000, adj_tolerance=20)‘ -l 1 --input-file $TPDATA/tp_dataset_run020472_2s.txt json
-```
+**Offline live emulation currently not supported in v5**
 
 ## Development
 * [Development workflow](https://dune-daq-sw.readthedocs.io/en/latest/packages/daq-release/development_workflow_gitflow/)
-* [Setting up DUNE DAQ development area](https://github.com/DUNE-DAQ/daqconf/wiki/Instructions-for-setting-up-an-FD-development-software-area)
+* [Setting up DUNE DAQ development area (v5.3)](https://github.com/DUNE-DAQ/daqconf/wiki/Setting-up-a-fddaq%E2%80%90v5.3.0-development-area)
```

Comment:
This may be annoying to keep track of.

Reply:
Yeah. I've now changed the link to `https://github.com/DUNE-DAQ/daqconf/wiki`, so we hopefully won't have to keep track of it anymore.